src/mjolnir/Makefile

```diff
@@ -0,0 +1,1117 @@
+# CMAKE generated file: DO NOT EDIT!
```

Comment:
no cmake generated makefiles, guess a relict from your previous in-source build?

Reply:
this shouldn't have happened, as I deleted all files inside the previous build and re-cloned.